src/main/java/seedu/exercise/model/property/Date.java

```diff
@@ -1,11 +1,14 @@
 package seedu.exercise.model.property;
 
+import static java.time.temporal.ChronoUnit.DAYS;
 import static java.util.Objects.requireNonNull;
 import static seedu.exercise.commons.util.AppUtil.checkArgument;
 
 import java.time.LocalDate;
+import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
 
 /**
  * Represents an Exercise's date in ExerHealth.
```

Comment:
Change the documentation here to reflect what the date now does. It is no longer just `an Exercise's date`

Reply:
done